src/java/org/tensorics/core/tensor/MappableTensor.java

```diff
@@ -0,0 +1,40 @@
+package org.tensorics.core.tensor;
+
+import java.util.Map;
+
+/**
+ * Enhances the tensor interface by a method to view the tensor as a map from
+ * position to values. This interface is intended to be used by utility methods
+ * in order to avoid unnecessary calls.
+ * 
+ * @author kaifox
+ *
+ * @param <V>
+ *            the type of the values of the tensors
+ */
+public interface MappableTensor<V> extends Tensor<V> {

```

Comment:
What is the advantage of having MappableTensor extend Tensor, instead of Mappable being independent and ImmutableTensor implementing Tensor, Mappable?
Just to limit the number of levels in the class hierarchy, especially if we're going to introduce at some point other specialized interfaces like Sliceable, Sortable, etc. for optimization ...